### PR TITLE
Fix: query generator use proper values

### PR DESCRIPTION
**Pull request by xavierleune**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Licence       | Apache-2.0
| Fixed tickets | ¤

### CODE REVIEW

**Comment by xavierleune** [comment]
I plan to update the changelog once PR #40 has been merged

**Comment by xavierleune** [comment]
@vgrebccm everything looks fine now. Waiting for your review

**Review by vgrebccm — APPROVED** [review]
SGTM !

**Pull request merged**
